cg/test/management/contract.api.spec.js

```diff
@@ -160,15 +160,12 @@ describe('Event feed websocket', () => {
   });
 
   it('should emit events that occur on the blockchain in nicer format', async done => {
-    expect.assertions(5);
+    expect.assertions(4);
     socketSubscription.on('message', event => {
       const data = JSON.parse(event);
       expect(data.eventName).toEqual('Controller_ConsentGivenTo');
-      expect(data.params.subjectIdHash).toEqual(sha3('websocket-test'));
-      expect(data.params.newProcessorsWhiteListed).toHaveLength(1);
-      expect(data.params.newProcessorsWhiteListed[0].toLowerCase()).toEqual(
-        process.env.CONTRACT_OWNER_ADDRESS.toLowerCase()
-      );
+      expect(data.params.subjectId).toEqual(sha3('websocket-test'));
+      expect(data.params.processorsConsented).toHaveLength(0);
```

Comment:
Is it OK, that on `Controller_ConsentGivenTo` the `data.params.processorsConsented` array is empty?

Reply:
Actually it is not, because the controller should be in that array. If we call recordConsentGivenTo without any processors (with an empty array), it will still set the controller as consented. Will look into this

Reply:
Created a task to handle this, this PR is too big